srf-weather-connector/processes/srfWeatherForecast.p.json

```diff
@@ -0,0 +1,141 @@
+{
+  "format" : "10.0.0",
+  "id" : "18A074A7BF4A40CC",
+  "kind" : "CALLABLE_SUB",
+  "config" : {
+    "data" : "com.axonivy.connector.srf.weather.connector.srfWeatherForecastData"
+  },
+  "elements" : [ {
+      "id" : "f0",
+      "type" : "CallSubStart",
+      "name" : "call(Number,String)",
+      "config" : {
+        "callSignature" : "call",
+        "input" : {
+          "params" : [
+            { "name" : "zip", "type" : "Number" },
+            { "name" : "locationName", "type" : "String" }

```

Comment:
As you already marked your call sub start as a connector, it now appears in the connector palette -> very nice!
You can help adopters of your connector even more if you add descriptions to this start and its parameters.